# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
# Result
atlas_msgs/AtlasSimInterfaceState end_state
bool success
